### ISSUE TITLE
missing forms after update to 1.5.2,  

### ISSUE THREAD

**Issue by Csakirobert** [13pixlar/sticky-list#25]
clicking on the edit or view do not show the form at all.

**Comment by 13pixlar** [comment]
I will invesrigate. What version of GF and WP are you using?

**Comment by Csakirobert** [comment]
GF: 2.4.5

WP 5.0.1


2019. 02. 09. 18:34 keltezéssel, 13pixlar írta:

**Comment by 13pixlar** [comment]
Could me wp 5 that's causing it. Only tested up to 4.9.9. Would it be
possible to do a quick wp downgrade and test?

Mvh

Adam

Den sön 10 feb. 2019 15:21Csakirobert <notifications@github.com> skrev:

**Comment by Csakirobert** [comment]
HI!

Sorry, I can't downgrade now.

Robert


*dr. Csáki Róbert *
Ügyvezető
Tel.:(+36)70/944-1387 <tel:%28%2B36%2970%2F944-1387>
www.antamsziro.hu <http://www.antamsziro.hu>
www.antamsziro.hu
2019. 02. 10. 15:51 keltezéssel, 13pixlar írta:

**Comment by jaakkorousu** [comment]
Yep same here with 5.0.3. It also renders all navigation drop down menus unusable (basic nav menus for the site).

Can I get the older version of sticky list somewhere?

**Comment by Ilardo** [comment]
Hi, also for me the same. There is no form to edit.

**Comment by 13pixlar** [comment]
I have now tested with WordPress 5.5+ and it seems to be working. Can any of you confirm?

Note that the plugin folder needs to be called `gravity-forms-sticky-list`

**Comment by Csakirobert** [comment]
It works for me with Wordpress 5.5.3


2020. 12. 04. 13:39 keltezéssel, 13pixlar írta:
